doc/user-guide/How-to-build.md

```diff
@@ -68,13 +68,10 @@ $ brew install erlang openssl unixodbc
 Add OpenSSL paths to the compiler and linker environment variables:
 
 ```bash
-$ export LDFLAGS="-L/usr/local/Cellar/openssl/[installed version]/lib/ -undefined dynamic_lookup $LDFLAGS"
-$ export CFLAGS="-I/usr/local/Cellar/openssl/[installed version]/include/ $CFLAGS"
+$ export LDFLAGS="-L/usr/local/opt/openssl/lib"
```

Comment:
Why did you remove the `-undefined dynamic_lookup`? Also your new export totally forgets about existing `LDFLAGS` which might be set.